CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+* Implements support for developers to disable sign up on multiple providers in
+FirebaseUI by defining an adminRestrictedOperation config and adding support in
```

Comment:
This has to be a single line. Change to something more understandable and less technical:
` * Improved UI support for all identity provider sign up operations when they are restricted by site administrators.`